.azure-pipelines-templates/matrix.yml

```diff
@@ -16,7 +16,7 @@ parameters:
     CIMetrics:
       container: nosgx
       pool: Ubuntu-1804-D8s_v3
-      dependsOn: ['BuildAndTest_SGX_CFT_FTCP_Perf', 'BuildAndTest_SGX_BFT_FTCP_Perf']
+      dependsOn: ['BuildAndTest_SGX_CFT_FTCP_Perf', 'BuildAndTest_SGX_BFT_FTCP_Perf', 'BuildAndTest_SGX_CFT_HTTP_Perf', 'BuildAndTest_SGX_BFT_HTTP_Perf']
```

Comment:
I'd really like to find a way to generate that from the perf test generation loop, but I really haven't found any way. Suggestions welcome.